huebot/README.md

```diff
@@ -13,3 +13,20 @@ Control your [Philips Hue Lights](www2.meethue.com/) using [Slack](https://api.s
 ## Created by
 - [Tobias Büschel](https://github.com/tobiasbueschel)
 - [Luke Jackson](https://github.com/lukejacksonn)
+
+
+
+##Troubleshooting
+
+Ensure Raspberry Pi is turned on and connected to the Internet
+
+Run `node ~/Desktop/cs50/huebot/index.js`
+
+In a new Terminal window, run `ngrok http 8080`
+[img](http://imgur.com/nzN9PLC)
```

Comment:
Have a look at the syntax for embedding images.. here https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images